bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -204,11 +212,12 @@ private void registerForLifecycleEvents(@NonNull Context context) {
                 new ActivityLifecycleCallbacksAdapter() {
                     @Override
                     public void onActivityCreated(Activity activity, Bundle savedInstanceState) {
+                        mIsClearAllowed = true;
+                        mIsConfigChange = false;
+
```

Comment:
You're missing the ` mIsFirstCreateCall = false;` inside the `openedFromDeeplink` check:

```java
    if (openedFromDeeplink(activity)) {
       mIsFirstCreateCall = false;
       return;
    }
```

I'm pretty sure we're going to need that.